app/code/Magento/PageBuilder/Model/Wysiwyg/DefaultConfigProvider.php

```diff
@@ -42,7 +42,7 @@ public function getConfig(\Magento\Framework\DataObject $config): \Magento\Frame
         $config->addData(
             [
                 'tinymce' => [
-                    'toolbar' => 'undo redo | styleselect | fontsizeselect | lineheight | forecolor backcolor ' .
+                    'toolbar' => 'undo redo | styles | fontsizeselect | lineheight | forecolor backcolor ' .
```

Comment:
@rhoerr do you want to use the same toolbar we just merged or this one is ok?

Reply:
Yes, I like that idea. We should make that a separate PR though so this stays just the upstream changes.

Reply:
correct